pkg/reboot/calculator.go

```diff
@@ -98,16 +98,15 @@ func (s *safeTimeCalculator) calcNumOfBatches() int {
 
 	var numberOfBatches int
 	switch {
-	// 3 workers of first batch + 10% for each batch
-	case workerNodesCount >= MaxBatchesAfterFirst+NodesNumberInFirstBatch:
+	//high number of workers we need max batches (for example 53 nodes will be done in 11 batches -> 1 * 3 + 10 * 5 )
```

Comment:
`high number of workers we need max batches` -> `for high number of workers we need max batches`

Reply:
I'd also rephrase this a little, or even just `high number of workers: we need max batches`